FAQ.md

```diff
@@ -0,0 +1,46 @@
+SignalR Frequently Asked Questions
+=======
+
+This document will condense some of the more popular questions and themes that have come up in the issues we've received on SignalR. Please check the FAQ before you submit a new issue, as the issue may have already been addressed below. 
+
+---
+
+**What is SignalR?**
+
+SignalR is a series of libraries for both the server and client that make it incredibly simple to add real-time web functionality to your apps. You can learn more about SignalR in the [SignalR documentation](https://docs.microsoft.com/en-us/aspnet/core/signalr/introduction). 
+
+**How does SignalR enable "real-time HTTP?"**
+
+There are two sets of open-source libraries that make this possible: 
+
+1. The server-side package, which is used by .NET developers to build server-side SignalR Hubs. This package is distributed via [NuGet](https://nuget.org) in the [Microsoft.AspNetCore.SignalR](https://www.nuget.org/packages/Microsoft.AspNetCore.SignalR) package. This server-side package is not required if your applications make use of the [SignalR Azure Service](https://azure.microsoft.com/en-us/services/signalr-service/). 
```

Comment:
Users shouldn't see the server-side package at all unless they are using .NET Framework. I'd just say it's distributed as part of ASP.NET Core (in the `Microsoft.AspNetCore.App` package)

Reply:
@anurse made changes to your much-appreciated feedback. Let me know if you have more to add, looking forward to it.